cpp/validate_harvested_records.cc

```diff
@@ -76,35 +77,85 @@ struct FieldInfo {
 };
 
 
-class JournalInfo {
-    std::string zeder_id_;
-    std::string zeder_instance_;
-    bool not_in_database_yet_;
+struct GeneralInfo {
     std::vector<FieldInfo> field_infos_;
```

Comment:
If that is the only field I have serious doubts about the utility of this struct!

Reply:
... see previous discussion